src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1331,17 +1510,194 @@ GlobeControls.prototype.moveTarget = function moveTarget() {
     return movingCameraTargetOnGlobe;
 };
 
-GlobeControls.prototype.pan = function pan(deltaX, deltaY) {
-    this.mouseToPan(deltaX, deltaY);
+/**
+ * Displaces the central point to a specific amount of pixels from its current position.
+ * The view flies to the desired coordinate, i.e.is not teleported instantly. Note : The results can be strange in some cases, if ever possible, when e.g.the camera looks horizontally or if the displaced center would not pick the ground once displaced.
+ * @param      {vector}  pVector  The vector
+ */
+GlobeControls.prototype.pan = function pan(pVector) {
+    this.mouseToPan(pVector.x, pVector.y);
     this.updateCameraTransformation(CONTROL_STATE.PAN);
+    this.dispatchEvent(this.changeEvent);
+    return waitSceneLoaded().then(() => {
+        this.updateCameraTransformation();
+    });
+};
+
+/**
+ * Returns the orientation angles of the current camera, in degrees.
+ * <!--<iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/okfj460p/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>-->
+ */
+GlobeControls.prototype.getCameraOrientation = function getCameraOrientation() {
+    var tiltCam = this.getTilt();
+    var headingCam = this.getHeading();
+    return [tiltCam, headingCam];
+};
+
+/**
+ * Returns the camera location projected on the ground in lat,lon.
+ * <!--<iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/mjv7ha02/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>-->
+ * @return {Position} position
+ */
+
+GlobeControls.prototype.getCameraLocation = function getCameraLocation() {
+    return C.fromXYZ('EPSG:4978', this.camera.position).as('EPSG:4326');
```

Comment:
`C.fromXYZ`  doesn't exist. You can use `new Coordinates('EPSH:4978', this.camera.position)` instead